PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,11 @@
 What does this PR do?
 
 #### Any relevant tickets
+Please use the phrase 'Closes' if it will fully complete the ticket.
 
 #### How has this been tested?
-- [ ] Unit tests
-- [ ] Integration tests
+Please indicate the need for this change to be tested. If not please justify, if so then at what levels. **Reviewers** ensure that these test requirements are also reviewed.
+
+- [ ] Unit / Integration tests
 - [ ] Browser tests
-- [ ] Review
-- [ ] Styleguide
+- [ ] End2end tests
```

Comment:
I've never written an end to end test myself (as opposed to our browser tests). Where are these written?

If it's in another codebase, then does "End2end" need to be part of the checklist for a PR that's going into this codebase?